src/controllers/trips.controller.js

```diff
@@ -37,6 +37,51 @@ class tripsController {
       );
     }
   }
+
+  /**
+   * user will recieve all requests he/she  made and data is retrieved from the database
+   * @param {Object} req The request object
+   * @param {Object} res The response object
+   * @returns {Object} A user object with selected fields
+   */
+  static async returnTripController(req, res) {
```

Comment:
Hey @Jajabenit250, the function should not have -controller suffix in their names. Besides the function is inside a controller, is there a reason to add -controller to the name?